src/image_copy/main.go

```diff
@@ -69,44 +73,59 @@ func getTags(dstTag string, updateMajor bool) []string {
 	return tags
 }
 
+func execCmd(name string, args ...string) string {
+	cmd := exec.Command(name, args...)
+	fmt.Println("\n> ", name, strings.Join(args, " "))
+	out, err := cmd.CombinedOutput()
+	if err != nil {
+		fmt.Println(out)
```

Comment:
do you need a string() here? sometimes go likes to dump []byte as a list of integers

Reply:
Right; good catch!